sass/_twitter-night-mode.scss

```diff
@@ -325,6 +325,11 @@ body.logged-in {
 	background-color: $dark-bg!important;
 }
 
+.DashUserDropdown.dropdown-menu li>a,
+.DashUserDropdown.dropdown-menu li>button {
+	color: $tweet-text-color;
```

Comment:
Can you change this to $white? It would make more sense to not be tied to whatever $tweet-text-color is for.